data/db_demo.yaml

```diff
@@ -93,6 +98,15 @@ telescope:
     skycam_link: /static/images/kcam_2.jpg
     robotic: false
     =id: Keck1
+  - name: LT Telescope
+    nickname: LT
+    lat: 28.762
+    lon: -17.872
+    elevation: 2363
+    diameter: 2.0
+    skycam_link: http://catserver.ing.iac.es/weather/archive/concam/concam_labels.png
```

Comment:
Looks like this link is stuck on 20200817, is there a more frequently refreshing link?